<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-UU5MJ9K" failures="1" tests="2" name="Xero Login" time="21.519" errors="0" timestamp="2021-09-01T19:13:35 NZST">
  <testcase classname="com.xeroFrameworkNikhat.testcases.BaseTest" name="@AfterMethod QuitAppMethod" time="0.432">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.xeroFrameworkNikhat.testcases.BaseTest.QuitAppMethod(BaseTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod QuitAppMethod -->
  <testcase classname="com.xeroFrameworkNikhat.testcases.LogintoXeroValidAccount" name="loginToAccount" time="14.818"/>
  <testcase classname="com.xeroFrameworkNikhat.testcases.HomePageXero" name="logIn" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.xeroFrameworkNikhat.testcases.HomePageXero.logIn(HomePageXero.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- logIn -->
  <testcase classname="com.xeroFrameworkNikhat.testcases.BaseTest" name="@BeforeSuite setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase classname="com.xeroFrameworkNikhat.testcases.BaseTest" name="@AfterSuite QuitApp" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite QuitApp -->
</testsuite> <!-- Xero Login -->
